projects/canopy/src/lib/skeleton/skeleton.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, ElementRef, Renderer2, Input, HostBinding } from '@angular/core';
+
+@Directive({
+  selector: '[lgSkeleton]',
+})
+export class LgSkeletonDirective {
+  @Input('lgSkeleton') lgSkeletonWidth = 4;
```

Comment:
I think we should also have a way of setting the `height` as well. 
For example, this was the design of one of our skeletons:
![Screenshot 2021-04-09 at 09 46 26](https://user-images.githubusercontent.com/8397116/114154658-8b48c900-9918-11eb-835e-f228c9767611.png)

Reply:
It looks like the styles would take up the height of the element applied to it. Are you thinking there could be instances where you need to override the height? Maybe inline elements would need it more 🤔

Reply:
If I have and `h1` I wouldn't set an height on that element because it would be handled by the font-size of the text. If there is no text (because it's being fetched) there is no font to resize and the h1 would basically inexistent. I'll put a screenshot of what I mean because I'm not sure I explained it very well x)

With text:
![Screenshot 2021-04-09 at 10 31 05](https://user-images.githubusercontent.com/8397116/114160371-bfbf8380-991e-11eb-9a83-952017d869a7.png)

Without text:
![Screenshot 2021-04-09 at 10 31 18](https://user-images.githubusercontent.com/8397116/114160392-c5b56480-991e-11eb-8614-83f632bf8a98.png)

Reply:
This is what I was trying to set automatically using the pseudo `:after` element, it was an experiment more than anything. Will be interesting to see if @paul-request managed to take it anywhere.